youtube_dl/extractor/malltv.py

```diff
@@ -0,0 +1,65 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import parse_duration
+
+
+class MallTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mall\.tv/(?:.+/)?(?P<id>.+)'
+    _TESTS = [
+        {
+            'url': ('https://www.mall.tv/18-miliard-pro-neziskovky-opravdu-jsou-sportovci-nebo-clovek-v-tisni-pijavice'),
+            'info_dict': {
+                'id': ('18-miliard-pro-neziskovky-opravdu-jsou-sportovci-nebo-clovek-v-tisni-pijavice'),
+                'ext': 'mp4',
+                'title': ('18 miliard pro neziskovky. Opravdu jsou sportovci nebo Člověk v tísni pijavice?'),
+                'description': ('Pokud někdo hospodaří s penězmi daňových '
```

Comment:
md5.

Reply:
This I find very complicated. I've originally got it here, but then after a while without any change on my side the tests failed due to different hash. So after some digging I fuond they changed a video (maybe a commercial or whatever).
What should be the procedure here then?

Reply:
Procedure for what? Once description changes test will fail regardless of whether it's an md5 or complete description.

Reply:
Then I don't understand necessity of md5 other than try tu run the test and don't care for outcome.

Reply:
Removing useless noise.

Reply:
done